### PR TITLE
Build against all numpys greater than 1.8.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,13 +29,13 @@ requirements:
     - ipopt
     - setuptools
     - cython
-    - numpy x.x
+    - numpy 1.8.*
```
Comment (isuruf) [inline recipe/meta.yaml]: Only numpy `1.11` is available across all platforms and python versions in conda-forge.

Comment (moorepants) [reply recipe/meta.yaml]: This only builds for macosx and linux. From the docs it seemed like this should work.

**Comment by moorepants** [comment]
The issue I'm facing is that I have another package that requires numpy > 1.13 and this has never been built for numpy 1.13. It seems like we need something that automatically builds packages with `numpy x.x` everytime  a new numpy version comes out.

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/meta.yaml]: Yes, but scipy is built against numpy 1.9, so you need at least 1.9 here

**Pull request merged**